GLAA.Services/Automapper/AlternativeBusinessRepresentativeProfile.cs

```diff
@@ -11,7 +11,7 @@ public class AlternativeBusinessRepresentativeProfile : Profile
         public AlternativeBusinessRepresentativeProfile()
         {
             CreateMap<Licence, AlternativeBusinessRepresentativeCollectionViewModel>()
-                .ForMember(x => x.AlternativeBusinessRepresentatives, opt => opt.MapFrom(y => y.AlternativeBusinessRepresentatives))
+                .ForMember(x => x.AlternativeBusinessRepresentatives, opt => opt.MapFrom(y => y.AlternativeBusinessRepresentatives.Where(z => !z.Deleted)))
```

Comment:
Is this technique going to mean we aren't able to map DB entities which are deleted to viewmodels? Although I can't think of a specific need, I have a feeling when we delve into the audit stuff we may need to show things like when entites are deleted.
Take for example a licence where someone has deleted an ABR, we'd want to be able to show admins the action taken and allow them to see the data which was removed.

Reply:
Yeah, it's a bit of a dirty way to do things. I wanted to be sure that the current site outwardly behaves the same but I think the repository methods should filter out deleted records before they get to the mapping stage. I'll take this filtering out.

Reply:
Yeah i suspected that was the case and I fully understand the reasoning! I think the mapping should avoid making logical decisions, mainly cos I don't want to create a second set of mappings (I think I would cry! :P)

Reply:
Ah, I remember why this was here now. The deleted status isn't in the view models so deleted records are filtered out here instead of in the views. I wanted to get this out the door quickly so I could get back on the form restructure which is holding up more stuff. Happy to move it to the VMs if you prefer, should only take an hour or two.